compiler-rt/lib/sanitizer_common/sanitizer_mutex.h

```diff
@@ -163,7 +163,11 @@ class SANITIZER_MUTEX Mutex : CheckedMutex {
   explicit constexpr Mutex(MutexType type = MutexUnchecked)
       : CheckedMutex(type) {}
 
+  // This mutex uses a 64-bit state value, meaning 64-bit atomic operations.
```

Comment:
add a `CHEERP: ` at the beginning of the comment, to clarify that we wrote it

Reply:
Done.